### ISSUE TITLE
`show  @@datanode` get error results when `drop database` executing error

### ISSUE THREAD

**Issue by FlyingMao** [actiontech/dble#1721]
* **dble version：**  
dble-9.9.9.9-a4c50ad7c7951a59e93ab1970a43f946efab4d98-20200319052026
* **preconditions ：**  
```
mysql> create table sharding_1_t2(id int);
Query OK, 0 rows affected (0.01 sec)
```
* **configs：** 

**schema.xml**  

```
<schema name="schema1" sqlMaxLimit="100" dataNode="dn1">
		<table name="sharding_1_t2" cacheKey="id" dataNode="dn1" />
               <table name="sharding_4_t1" dataNode="dn1,dn2,dn3,dn4" rule="hash-four" />
</schema>		
    <dataNode name="dn1" dataHost="10.186.60.41" database="db1"/>
    <dataNode name="dn2" dataHost="10.186.60.42" database="db1"/>
    <dataNode name="dn3" dataHost="10.186.60.41" database="db2"/>
    <dataNode name="dn4" dataHost="10.186.60.42" database="db2"/>

    <dataHost balance="0" maxCon="1000" minCon="10" name="10.186.60.41" switchType="2" slaveThreshold="100">
        <heartbeat>show slave status</heartbeat>
        <writeHost host="hostM1" url="10.186.60.41:3306" password="test" user="test">
        </writeHost>
    </dataHost>
    <dataHost balance="0" maxCon="1000" minCon="10" name="10.186.60.42" switchType="2" slaveThreshold="100">
        <heartbeat>show slave status</heartbeat>
        <writeHost host="hostW1" url="10.186.60.42:3306" password="test" user="test"/>
    </dataHost>

```

**rule.xml**  

```


```

**server.xml**  

```


```

* **steps：**    
    step1.  start a transaction to make datanode:dn1 used in dble
```
mysql> begin;
Query OK, 0 rows affected (0.00 sec)

mysql> insert into sharding_1_t2 values(1);
Query OK, 1 row affected (0.00 sec)
```

step2. execute `drop database` in dble manager port util return error
```
mysql> drop database @@datanode='dn$1-3';
ERROR 1105 (HY000): Occur error in [dn1]
```

step3. check resullt of `show @@datanode;`
```
mysql> show @@datanode;
```
* **expect result：**  
    1.
```
mysql> show @@datanode;
+------+------------------+---------------+-------+--------+------+------+---------+---------------+
| NAME | DATHOST          | SCHEMA_EXISTS | INDEX | ACTIVE | IDLE | SIZE | EXECUTE | RECOVERY_TIME |
+------+------------------+---------------+-------+--------+------+------+---------+---------------+
| dn1  | 10.186.60.41/db1 | true         |     0 |      0 |    1 | 1000 |       8 |            -1 |
| dn2  | 10.186.60.42/db1 | false         |     0 |      0 |    1 | 1000 |       4 |            -1 |
| dn3  | 10.186.60.41/db2 | false         |     0 |      0 |    1 | 1000 |       4 |            -1 |
| dn4  | 10.186.60.42/db2 | true          |     0 |      0 |    3 | 1000 |      10 |            -1 |
+------+------------------+---------------+-------+--------+------+------+---------+---------------+
```
* **real result：**  
    1.
```
mysql> show @@datanode;
+------+------------------+---------------+-------+--------+------+------+---------+---------------+
| NAME | DATHOST          | SCHEMA_EXISTS | INDEX | ACTIVE | IDLE | SIZE | EXECUTE | RECOVERY_TIME |
+------+------------------+---------------+-------+--------+------+------+---------+---------------+
| dn1  | 10.186.60.41/db1 | false        |     0 |      0 |    1 | 1000 |       8 |            -1 |
| dn2  | 10.186.60.42/db1 | false         |     0 |      0 |    1 | 1000 |       4 |            -1 |
| dn3  | 10.186.60.41/db2 | false         |     0 |      0 |    1 | 1000 |       4 |            -1 |
| dn4  | 10.186.60.42/db2 | true          |     0 |      0 |    3 | 1000 |      10 |            -1 |
+------+------------------+---------------+-------+--------+------+------+---------+---------------+
```
* **supplements：**  
    1. check whether `10.186.60.41/db1` really exists in mysql
```
mysql> show databases;
+--------------------+
| Database           |
+--------------------+
| information_schema |
| db1                |
| db3                |
| db4                |
| mysql              |
| performance_schema |
| sys                |
+--------------------+
7 rows in set (0.00 sec)
```

2.[dble.log](https://github.com/actiontech/dble/files/4358251/dble.log)

**Comment by Baofengqi** [comment]
please provide dble.log

**Comment by FlyingMao** [comment]
added

**Comment by Baofengqi** [comment]
I think it is not a bug.

**Comment by FlyingMao** [comment]
Accepted as a known defects,but hope to tuning the error message of `drop`

**Comment by FlyingMao** [comment]
resolved version: dble-9.9.9.9-9bf5140bf8483248e94077ba0b429e7d4acfb353-20200326102553
